### PR TITLE
Handsfree radiation collector array.

### PR DESCRIPTION
**Pull request by Francisco-Franks**
[Engineering]

## About the PR 

This allows tanks to be click-dragged to the radiation collector array.

## Why's this needed? 
My friend Nuggets the Chief Engineer, who lost all their limbs, wants to make the best singularity. Would you really deny Nuggets their destiny?

## Changelog 

```changelog
(u)MetaReferencer
(+)You can click-drag plasma tanks onto radiation collectors.
```

### CODE REVIEW

**Comment by 444Portal** [comment]
Tagged with Silicons as that is what the original PR that was later adapted into this was aimed at.

My personal thoughts:
Despite the fact that I did say I would prefer a click drag, this seems odd. I believe the general consensus was that this is not a good idea, as Cyborgs should not be able to do anything. Given that this PR, which accomplishes the exact same thing as the prior one, has Cyborgs missing from any of the descriptions, it seems even more odd.

If this is not aimed at Cyborgs, why do we need a click-drag?
If this is aimed at Cyborgs, do they really need to be able to do this? As previously discussed, you can call a human over often. Even then, Cyborgs have no issue setting up solars which does the job in most cases.

**Comment by Francisco-Franks** [comment]
Part of Leah's argument was "TONGS, FOR A SPECIFIC ITEM, FOR BORGS" so I am eliminating this. This is exactly on parity with, say, borgs being able to do surgery via clickdrag. It also allows a hypothetical armless human to do their job. This doesn't allow interaction with anything else other than the radiation collector. It is as hyperfocused as it can get, borgs can't cool the plasma or get cool plasma, they can use the most basic available plasma tank and that is literally it.
In short, there is precedent (in the surgery stuff), and it allows a borg to set up the most basic power gen and that is all.

**Comment by TobleroneSwordfish** [comment]
There is precedent, but the precedent is IMO bad design and shouldn't be used as an argument to keep adding clickdrag interactions to everything borgs can't do.

**Comment by Francisco-Franks** [comment]
Twas worth a shot.
